lndclient/lightning_client.go

```diff
@@ -36,6 +37,11 @@ type LightningClient interface {
 
 	AddInvoice(ctx context.Context, in *invoicesrpc.AddInvoiceData) (
 		lntypes.Hash, string, error)
+
+	SignMessage(ctx context.Context, msg []byte) (string, error)
+
+	VerifyMessage(ctx context.Context, msg []byte, sig string) (bool,
+		*btcec.PublicKey, error)
```

Comment:
It is pre-existing (for the other fields), but we normally document external methods.